google/ggrc-core
# GGRC-6801 Related information popup now shows mapped object type

# Issue description

Related information popup shows Control instead of Access group.

# Steps to test the changes

Login to GGRC application:

Open any audit page
Map control and access group to audit
Create assessment with Control type
Open Assessment info panel> Other attributes tab
Click Map icon for Related Information and map Access group snapshot to assessment
Click on Access group snapshot to open popup
Actual Result: Related information popup shows Control instead of Access group
Expected Result: Related information popup should show mapped object type
Look at the DESCRIPTION

# Solution description

Code changes done to make sure related information popup shows the mapped object type.

# Sanity checklist




# PR Review checklist
